frontend/package.json

```diff
@@ -3,7 +3,6 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@kubeflow/frontend": "github:kubeflow/frontend#7c10d414e5f445aba4c057aecacb240b0f21a7bb",
```

Comment:
Did you run `npm rm -S @kubeflow/frontend`?
I think package-lock.json should also have the same change

Reply:
Thank you for the reference! Updated package-lock.json accordingly to reflect the changes.